footprints/main/models.py

```diff
@@ -994,8 +969,15 @@ def display_title(self):
             return self.title
 
     def owners(self):
-        role = Role.objects.get_owner_role()
-        return self.actor.filter(role=role)
+        return self.actor.filter(
+            role__name=Role.OWNER).select_related('person')
+
+    def owners_list(self):
+        a = ''
+        for actor in self.actor.all():
+            if actor.role.name == Role.OWNER:
+                a += actor.display_name() + ', '
+        return a[:-2]
```

Comment:
`.owners()` is not inherently inefficient when called for a single footprint. When called in a loop as a sorting key for all footprints, as it is here, there's a cost. I will work out a more palatable approach from a code perspective.